Core/src/org/sleuthkit/autopsy/centralrepository/contentviewer/OtherOccurrencesFilesTableModel.java

```diff
@@ -107,9 +109,9 @@ public Object getValueAt(int rowIdx, int colIdx) {
      *
      * @param newNodeData data to add to the table
      */
-    void addNodeData(OtherOccurrenceNodeData newNodeData) {
-        String newNodeKey = createNodeKey((OtherOccurrenceNodeInstanceData) newNodeData);//FilenameUtils.getName(((OtherOccurrenceNodeInstanceData)newNodeData).getFilePath());
-        List<OtherOccurrenceNodeData> nodeList = nodeMap.get(newNodeKey);
+    void addNodeData(NodeData newNodeData) {
+        String newNodeKey = createNodeKey((NodeData) newNodeData);//FilenameUtils.getName(((OtherOccurrenceNodeInstanceData)newNodeData).getFilePath());
```

Comment:
Redundant cast which was since the argument for addNodeData is now a NodeType that createNodeKey accepts.